Robot_Adapter/CRUD/Update/IUpdate.cs

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using BH.oM.Base;
+using BH.oM.Structure.Elements;
+using BH.oM.Structure.SectionProperties;
+using BH.oM.Structure.SurfaceProperties;
+using BH.oM.Structure.Constraints;
+using BH.oM.Structure.Loads;
+using BH.oM.Physical.Materials;
+using BH.oM.Adapter;
+using RobotOM;
+
+namespace BH.Adapter.Robot
+{
+    public partial class RobotAdapter
+    {
+        /***************************************************/
+        /****           Protected Methods               ****/
+        /***************************************************/
+
+        protected override bool IUpdate<T>(IEnumerable<T> objects, ActionConfig actionConfig = null)
+        {
+            bool success = true;
+            success = Update(objects as dynamic);
+            updateview();
+            return success;
+        }
+
+        /***************************************************/
+
+        protected bool Update(IEnumerable<IBHoMObject> bhomObjects)
+        {
+            return true;
```

Comment:
Good idea. How about `"Objects of type '" + bhomObjects.First().GetType().ToString() + "' not updated"`

Reply:
Something like that would work I think. Might tweak it slightly (to avoid crashes for empty collections) to:

```
IBHoMObject first = bhomObjects.FirstOrDefault();

if(first != null)
{
    Engine.Reflection.Compute.RecordWarning("Objects to type " + first.GetType().Name + " can currently not be updated");
}

```

WIll let you choose the exact wording, but good to use FirstORDefault followed by a null check! :)